docs/rules/no-process-exit.md

```diff
@@ -23,3 +23,18 @@ process.on('SIGINT', () => {
     process.exit(1);
 });
 ```
+
+```js
```

Comment:
Could this example be simpler?

Reply:
Yes. I could just import the module, comment the .then case block as "// do stuff". I'm on it